internal/pkg/workspace/workspace.go

```diff
@@ -476,7 +477,7 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 				continue
 			}
 			fname := f.Name()
-			if strings.Contains(strings.ToLower(fname), dockerfileName) {
+			if strings.Contains(strings.ToLower(fname), dockerfileName) && !strings.Contains(strings.ToLower(fname), dockerignoreName) {
```

Comment:
Do we need to change it here as well?
```suggestion
			if strings.Contains(strings.ToLower(fname), dockerfileName) && !strings.HasSuffix(strings.ToLower(fname), dockerignoreName) {
```